app/controllers/api/v1/patients_controller.rb

```diff
@@ -23,13 +22,13 @@ def sync_to_user
       if patients_to_sync.empty?
         processed_since
       else
-        patients_to_sync.last.updated_on_server_at
+        patients_to_sync.last.updated_at
       end
 
     render(
       json:   {
         patients:        patients_to_sync.map(&:nested_hash),
-        processed_since: most_recent_record_timestamp
+        processed_since: most_recent_record_timestamp.strftime('%FT%T.%3NZ')
```

Comment:
We may want to make that format string a constant up above; makes the code more legible and maintainable